site/_posts/2020-09-23-jerry-hopper-fusionauth.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: blog-post
+title: Jerry loved FusionAuth so much he built an ARM image for it
+description: Community member Jerry Hopper discusses how and why he sold FusionAuth to his colleagues.
```

Comment:
Community member Jerry Hopper discusses how he discovered FusionAuth and how it's helped him and his teams save development time and authentication headaches.